lib/commands/plugin-test.sh

```diff
@@ -33,11 +35,32 @@ plugin_test_command() {
         fail_test "list-all did not return any version"
     fi
 
-    (asdf install $plugin_name ${versions[0]})
+    latest_version=${versions[0]}
```

Comment:
This won't be the latest version but the first version.

I think [line 28](https://github.com/asdf-vm/asdf/pull/61/files#diff-84d75d3e627bec07409965f77e402042R28) should read the following instead:

```sh
read -a versions <<< $(asdf list-all $plugin_name | tail -1)
```